Assets/Scripts/Models/PlayerModel.cs

```diff
@@ -0,0 +1,97 @@
+using UnityEngine;
+using Weapon;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace Player {
+
+  public class PlayerModel {
+    public static int MAX_WEAPON_CAPACITY = 10;
+    public static float MAX_HEALTH_POINT = 100;
+    public static float MAX_STAMINA_POINT = 100;
+    public enum Party { red, blue };
+
+    private string Name;
```

Comment:
Is this a unique identifier?

Reply:
I have not considered it yet. For now I just specified the name. But I can add a mechanism to assign an id to the player each time it is added.